version_number.txt

```diff
@@ -0,0 +1 @@
+1.8.0
```

Comment:
I wonder if there's a possibility of getting this from a git ref for the release version and then using something either random or static during development?  Maybe we can talk it through a bit in person.